src/Web/Views/Shared/_Footer.cshtml

```diff
@@ -7,7 +7,7 @@
     </div>
 
     <div class="column is-half">
-      <nav class="navbar is-transparent" role="navigation" aria-label="footer navigation">
+      <nav class="navbar is-transparent" role="navigation" aria-label="footer navigation"></nav>
```

Comment:
I think the closing tag needs to be placed after line 16. Notice the divs and unordered lists on the next few lines are marked as `navbar-end` and `navbar-item`, respectively. Those should be children of the `nav` tag.

https://bulma.io/documentation/components/navbar/

Reply:
Okay. I just modified it